hw/ip/entropy_src/doc/theory_of_operation.md

```diff
@@ -12,6 +12,8 @@ After a reset, the ENTROPY_SRC block will start up in boot-time / bypass mode by
 This feature is designed to provide an initial seed's worth of entropy with lower latency than the normal FIPS/CC compliant health check process.
 Health testing will still be performed on boot-time mode entropy, but the window of checking is, by default, 384 bits instead of 2048 bits.
 When entropy is delivered to the downstream hardware block, a signal will indicate what type of entropy it is - FIPS/CC compliant or not.
+This signal is determined by the `FIPS_FLAG` field in the [`CONF`](registers.md#conf).
```

Comment:
Nit: I think you want "the CONF register" (currently missing the last word)

Reply:
You're right, thank you!